docs(nuget): explain authenticated feeds on Azure DevOps need right username (#29987)
renovatebot/renovate: docs/usage/nuget.md

}
```

If you use using Azure DevOps:

- you can set `matchHost` to `pkgs.dev.azure.com`
- remember to set the username, so Renovate can build the project when it creates the PR

<!-- prettier-ignore -->
!!! note
    Only Basic HTTP authentication (via username and password) is supported.

>     For Azure DevOps, you can use a PAT with `read` permissions on `Packaging` plus an empty username. (EDITOR NOTE: is the emtpy username correct????)

Reviewer: we should suggest to set the username to the user of the pat.

>     For Azure DevOps, you can use a PAT with `read` permissions on `Packaging` plus an empty username. (EDITOR NOTE: is the emtpy username correct????)

Author: ```suggestion
    For Azure DevOps: use a PAT with `read` permissions on `Packaging`.
    The username of the PAT must match the username of the _user of the PAT_.
```

How about this? Please check if this is technically correct.
